scripts/src/osd/sdl.lua

```diff
@@ -170,7 +170,11 @@ newoption {
 }
 
 if not _OPTIONS["NO_USE_XINPUT"] then
-	_OPTIONS["NO_USE_XINPUT"] = "1"
+	if _OPTIONS["targetos"]=="windows" or _OPTIONS["targetos"]=="macosx" or _OPTIONS["targetos"]=="haiku" or _OPTIONS["targetos"]=="asmjs" then
```

Comment:
I have to admit, this feels odd vs using a `targetos==linux` check. However, I decided it was best to exactly match the X11 enable-by-default check a few lines above.